internal/service/scaffold/options_test.go

```diff
@@ -0,0 +1,125 @@
+package scaffold_test
+
+import (
+	"io"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-project/modulectl/internal/service/scaffold"
+	iotools "github.com/kyma-project/modulectl/tools/io"
+)
+
+func Test_Validate_Options(t *testing.T) {
```

Comment:
Good point. Yes, the details of the dependency are usually not part of the SUT. But we keep testing the behavior that the service is passing on the error, since this is the logic that the SUT should do.

For the builder I am not sure, since it only exists to create valid options for the consumer (the service) to pass, because is can not be stubbed. The options unit test don't need that.

Reply:
ALso fine 👍🏻